cmd/dist/main.go

```diff
@@ -73,13 +73,15 @@ distribution tool
 		var (
 			debug   = context.GlobalBool("debug")
 			timeout = context.GlobalDuration("timeout")
+			cancel  contextpkg.CancelFunc
 		)
 		if debug {
 			logrus.SetLevel(logrus.DebugLevel)
 		}
 
 		if timeout > 0 {
-			background, _ = contextpkg.WithTimeout(background, timeout)
+			background, cancel = contextpkg.WithTimeout(background, timeout)
+			defer cancel()
```

Comment:
This will cancel the the context before the command can even run. It is not necessary to call cancel here.